front-end/src/Profile.css

```diff
@@ -10,7 +10,13 @@
   left: 30%;
   right: 30%;
 }
+#settings-title{
```

Comment:
I'm not sure what exactly is causing it, but on my screen the settings modal now has a black bar on its right side. You can see what I mean [in this image](https://cdn.discordapp.com/attachments/776234836456898602/825187738902200330/Screenshot_20210326_220056.png).

Reply:
I think it's supposed to be like that because it's a modal. It was even worse before-- the 'settings' was misaligned with the form--you can view it in master branch or I will attach screenshot later here

Reply:
I don't think that black bar on the right should be there. On master, for me, the modal currently looks [like this.](https://cdn.discordapp.com/attachments/776234836456898602/825354246290407444/Screenshot_20210327_090237.png)

Reply:
yes, on master, width of the 'settings' element was too large, so there misalignment between signup form and settings heading--i fixed that
but as you mentioned, there was also misalignment between react's modal (has default styling) and fieldset - i fixed that by changing fieldset to div hehe
screenshot is here: https://slack-files.com/T01LN2C2L20-F01SM4R4V2N-210aed3bb9